<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraConfigurationDialogue</class>
 <widget class="QDialog" name="CameraConfigurationDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/application/application.png</normaloff>:/application/application.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0">
    <widget class="QLabel" name="labelTurnRight">
     <property name="text">
      <string>Turn Right</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonResetKeyboardTurnSpeed">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButtonResetLeft">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelLeft">
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sliderMouseTurnSpeed">
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="SingleKeySequenceEdit" name="keyRight">
     <property name="keySequence">
      <string>D</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pushButtonResetUp">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="SingleKeySequenceEdit" name="keyBackward">
     <property name="keySequence">
      <string>S</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelMouseTurnSpeed">
     <property name="text">
      <string>Mouse Turn Speed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFieldOfView">
     <property name="text">
      <string>FOV</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelKeyboardTurnSpeed">
     <property name="text">
      <string>Keyboard Turn Speed</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="pushButtonResetDown">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelTurnLeft">
     <property name="text">
      <string>Turn Left</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="pushButtonResetRightTurn">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelRight">
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="SingleKeySequenceEdit" name="keyDown">
     <property name="keySequence">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="SingleKeySequenceEdit" name="keyForward">
     <property name="keySequence">
      <string>W</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButtonResetBackward">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="labelUp">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonResetMouseTurnSpeed">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="SingleKeySequenceEdit" name="keyUp">
     <property name="keySequence">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderKeyboardTurnSpeed">
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="SingleKeySequenceEdit" name="keyTurnLeft">
     <property name="keySequence">
      <string>Q</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderFieldOfView">
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="labelDown">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelMoveSpeed">
     <property name="text">
      <string>Move Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="sliderMoveSpeed">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelBackward">
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="pushButtonResetLeftTurn">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonResetMoveSpeed">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxAllowCameraEvents">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButtonResetRight">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelMouseControls">
     <property name="text">
      <string>Mouse Controls</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="SingleKeySequenceEdit" name="keyLeft">
     <property name="keySequence">
      <string>A</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="SingleKeySequenceEdit" name="keyTurnRight">
     <property name="keySequence">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBoxMouseControls">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButtonResetForward">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonResetMouseControlsEnabled">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelForward">
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonResetFieldOfView">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelAllowCameraEvents">
     <property name="text">
      <string>Allow Camera Events</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleKeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header>src/window/controls/SingleKeySequenceEdit/SingleKeySequenceEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
